<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettings</class>
 <widget class="QDialog" name="CSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>162</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QCheckBox" name="cbAsiLoader">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Using custom asi loader</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbWinMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Run game in window mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import servers</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>640*480</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>800*600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1024*768</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1280*1024</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1280*720</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1366*768</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1400*850</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1600*900</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
